src/jit/compiler.h

```diff
@@ -598,6 +598,20 @@ class  LclVarDsc
         return (unsigned)(roundUp(lvExactSize, sizeof(void*)));
     }
 
+    bool                lvIsMultiregStruct()
+    {
+#if FEATURE_MULTIREG_ARGS_OR_RET
+#ifdef _TARGET_ARM64_
+        if ((TypeGet() == TYP_STRUCT) &&
+            (lvSize()  == 2 * TARGET_POINTER_SIZE))
+        {
+            return true;
+        }
+#endif  // _TARGET_ARM64_
+#endif  // FEATURE_MULTIREG_ARGS_OR_RET
```

Comment:
We track both the exact size in lvExactSize and the aligned size via:

    unsigned            lvSize()        // Size needed for storage representation. Only used for structs
    {
        return (unsigned)(roundUp(lvExactSize, sizeof(void*)));
    }